tests/utils/test_github_project_queries.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2024 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from living_documentation_generator.utils.constants import (
+    PROJECTS_FROM_REPO_QUERY,
+    ISSUES_FROM_PROJECT_QUERY,
+    PROJECT_FIELD_OPTIONS_QUERY,
+    ISSUES_PER_PAGE_LIMIT,
+)
+from living_documentation_generator.utils.github_project_queries import (
+    get_projects_from_repo_query,
+    get_issues_from_project_query,
+    get_project_field_options_query,
+)
+
+
+# get_projects_from_repo_query
+
+
+def test_get_projects_from_repo_query():
```

Comment:
Can you add assert to check that in the query are not present not used {param} items? (In all tests.)

Reply:
Here I used this regres pattern `\{\w+\}`. As I found this is often used to find placeholders in a formatted string. See commit:

Reply:
See new related comment. This assert is not applied as format method inside the tested method behave in other way.